src/gui/tray/Window.qml

```diff
@@ -583,7 +602,7 @@ Window {
 
                     Text {
                         id: activityTextInfo
-                        text: (type === "Activity" || type === "File" || type === "Sync") ? displaypath : message
+                        text: (type === "Activity" || type === "Sync") ? displayPath : ((type === "File") ? subject : message)

```

Comment:
Just a suggestion, feel free to ignore, but I generally find chained ternary easier to read that way:
```
text: (type === "Activity" || type === "Sync") ? displayPath
    : (type === "File") ? subject
    : message
```